Content.Server/Zombies/ZombieSystem.Transform.cs

```diff
@@ -167,6 +167,9 @@ public void ZombifyEntity(EntityUid target, MobStateComponent? mobState = null)
                 tool.Qualities = new ("Prying");
                 tool.UseSound = new SoundPathSpecifier("/Audio/Items/crowbar.ogg");
                 Dirty(tool);
+
+                // allow prying open powered doors
+                EnsureComp<ToolForcePoweredComponent>(target);
```

Comment:
They shouldn't be relying on anything Tool related.

Reply:
As far as I can tell, this component is a misnomer. It was introduced by #10770 for the jaws of life specifically to allow prying of powered doors. It is not used anywhere else.

Reply:
If that's the case then it should be renamed with a different comment.